config/data/configs/single.node.config.toml

```diff
@@ -0,0 +1,83 @@
+[base]
+data_dir_path = "/var/folders/tq/767cwxb92czdjb84m1bkppch0000gn/T/6f41f887e9b84374ebb0d35bb04f5d33"
+
+[metrics]
+dir = "metrics"
+collection_interval_ms = 1000
+
+[execution]
+address = "localhost"
+port = 6183
+genesis_file_location = "genesis.blob"
+
+[admission_control]
+address = "0.0.0.0"
+admission_control_service_port = 8000
+need_to_check_mempool_before_validation = false
+max_concurrent_inbound_syncs = 100
+
+[admission_control.upstream_proxy_timeout]
+secs = 1
+nanos = 0
+
+[debug_interface]
+admission_control_node_debug_port = 6191
+storage_node_debug_port = 6194
+metrics_server_port = 9101
+public_metrics_server_port = 9102
+address = "localhost"
+
+[storage]
+address = "localhost"
+port = 6184
+dir = "libradb/db"
+grpc_max_receive_len = 100000000
+
+[[networks]]
+peer_id = "8deeeaed65f0cd7484a9e4e5ac51fbac548f2f71299a05e000156031ca78fb9f"
+listen_address = "/ip4/0.0.0.0/tcp/65206"
+advertised_address = "/ip4/0.0.0.0/tcp/65206"
+discovery_interval_ms = 1000
+connectivity_check_interval_ms = 5000
+enable_encryption_and_authentication = true
+is_permissioned = true
+role = "validator"
+network_keypairs_file = ""
+network_peers_file = ""
+seed_peers_file = ""
+
+[consensus]
+max_block_size = 100
```

Comment:
wait that's what we use, isn't that pretty low?

Reply:
/shrug/ I'm just the messenger, I suppose Andrey might be a good person to follow up with in terms of performance. I guess it would be nice to call this max_transactions_per_block instead... I'll clean that up in a follow up commit.